deploy/marlowe-runtime/templates/chain-indexer.yaml

```diff
@@ -4,9 +4,11 @@ apiVersion: core.oam.dev/v1beta1
 kind: Application
 metadata:
   name: chain-indexer-{{ $network }}
-  namespace: marlowe-staging
+  namespace: {{ $.Values.namespace }}
   annotations:
     app.oam.dev/publishVersion: {{ $.Chart.AppVersion }}
+    meta.helm.sh/release-name: {{ $.Values.releaseName }}
```

Comment:
Why are these needed? What do they mean?

Reply:
The deployments were failing without the annotations being present; seems they are for KubeVela to track Helm deployments.

Reply:
Is there any documentation on them? Is this why chart changes weren't always propagating?

Reply:
I don't think that is why chart changes weren't propagating. There are no documentations in Kubevela itself, but this SO answer captures an explanation for it: https://stackoverflow.com/a/69005327